app/src/main/java/to/dev/dev_android/view/main/view/MainActivity.kt

```diff
@@ -28,6 +29,15 @@ class MainActivity : BaseActivity<ActivityMainBinding>(), CustomWebChromeClient.
         setWebViewSettings()
         savedInstanceState?.let { restoreState(it) } ?: navigateToHome()
         handleIntent(intent)
+        PushNotifications.start(getApplicationContext(), "cdaf9857-fad0-4bfb-b360-64c1b2693ef3");
```

Comment:
Can this be extracted to an environment variable or something like it?

Reply:
I'll have to look into how to do it but it's a good idea